tests/zipkin_test.py

```diff
@@ -578,8 +578,31 @@ def test_stop_with_error(self):
             })
             assert len(span_storage) == 0
 
+    def test_error_stopping_log_context(self):
+        '''Tests if exception is raised while emitting traces that
```

Comment:
Can we do this as part of the integration test? So that it actually resembles what a user would do?